client/signup/jetpack-connect/main-wrapper.jsx

```diff
@@ -13,8 +13,9 @@ export default React.createClass( {
 	displayName: 'JetpackConnectMainWrapper',
 
 	render() {
+		const wrapperClassName = this.props.isWide ? 'jetpack-connect__main-wide' : 'jetpack-connect__main';
```

Comment:
I think we should consider doing something like `.jetpack-connect__main.is-wide` here. I feel like that more closely matches up with how we create classes in say notices where we have `is-error`, `is-success`, etc.

We could do that by changing this line to:

```javascript
const wrapperClassName = classNames( 'jetpack-connect__main' {
    'is-wide': this.props.isWide
} );
```